Getting Started with Form (#1485)
DevExpress/devextreme-documentation: concepts/05 UI Components/Form/01 Getting Started with Form/70 Validate the Form Data.md

DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different [validation rule types](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). 

To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/) to it.

> In the following example, we specify the [isRequired](Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property for the `Name` item. We also specify the **NumericRule** for `officeNumber` and **EmailRule** for `Email` items.

Reviewer: Maybe give links to the validation rules?

> An editor is validated when its value changes. To validate all editors simultaneously, call the [validate()](/Documentation/ApiReference/UI_Components/dxForm/Methods/#validate) Form method. 

Reviewer: This sentence looks out of place. Let's modify the Submit the Form topic a bit so that it includes validation before submission.

> DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different [validation rule types](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). 

Reviewer 2: The first sentence says "before they are saved". A sentence in the next paragraph says "an editor is validated when its value changes". I'm not sure which is right... Or maybe those two sentences talk about different things?

Author: The validation cycle is the following:
1. A user types/selects something.
2. The value changes.
3. The user moves the focus outside the editor or presses the 'Enter' button.
4. Validation checks trigger.
5. If the checks pass successfully, the value is saved, Otherwise, the red icon and error messages (if any) appear.

Reviewer 2: So then the following phrase in the other paragraph is wrong: "an editor is validated when its value changes". Right?

> To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/) to it.

Reviewer 3: ```suggestion
To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/).
```

> To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/) to it. An editor is validated when its value changes.

Reviewer 2: The last sentence in this paragraph probably belongs somewhere in the first paragraph. Looks like a common information about validation, not applied to the RequiredRule alone.

Author: Good point, I'll move this sentence to the first paragraph.

> In the following example, we specify the **isRequired** property for the `Name` item. We also specify the [NumericRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/NumericRule/) for `officeNumber` and [EmailRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/EmailRule/) for `Email` items.

Reviewer 2: ```suggestion
The following example sets the **isRequired** property for the `Name` item. It also adds the [NumericRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/NumericRule/) for `officeNumber` and [EmailRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/EmailRule/) for `Email`.
```

> In the following example, we specify the [isRequired](Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property for the `Name` item. We also specify the [NumericRule](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/NumericRule/) for `officeNumber` and [EmailRule](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/EmailRule/) for `Email` items.

Reviewer: > /Documentation/ApiReference/**UI_Widgets**/dxValidator/Validation_Rules/NumericRule/
/Documentation/ApiReference/**UI_Widgets**/dxValidator/Validation_Rules/EmailRule/

Outdated links
